### PR TITLE
Mcs regression tests

### PR DESCRIPTION
**Pull request by richardjgowers**
These are checked by hand to be sane.  I thought of writing fancy tests which match certain smarts in mol1 to certain other smarts in mol2, then thought these tests are probably enough for now...

### CODE REVIEW

File: test/test_mcs.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+
+def _rf(fn):
+    # get path to file from inside lomap installation
+    f = pkg_resources.resource_filename('lomap', 'test/' + fn)
+    return f.replace('/lomap/test', '/test')
+
+
+@pytest.fixture
+def toluene():
```
Comment (IAlibay) [inline test/test_mcs.py:20]: I'm not a massive fan of the way these are laid out - maybe they should be in conftest - but given it's how it was done before I won't block just for this.

File: test/test_mcs.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+@pytest.fixture
+def naphthol():
+    f = _rf('basic/2-naftanol.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+def test_toluene_to_methylcyclohexane(toluene, methylcyclohexane):
```
Comment (IAlibay) [inline test/test_mcs.py:75]: could these not all be squashed to a single parametrise? (or is it too unreadable?)

**Review by IAlibay — APPROVED** [review]

File: test/test_mcs.py (hunk 2 above)
Comment (mikemhenry) [reply test/test_mcs.py:75]: I also think parametrizing is the way to go here

File: test/test_mcs.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    assert sorted(mapping) == [(0, 0), (1, 1), (2, 6), (3, 5), (4, 4), (5, 3),
+                               (6, 2)]
+
+
+def test_toluene_to_methylnaphthalene(toluene, methylnaphthalene):
+    mapper = mcs.MCS(toluene, methylnaphthalene)
+
+    mapping = mapper.heavy_atom_mcs_map()
+
+    assert len(mapping) == 7
```
Comment (mikemhenry) [inline test/test_mcs.py:89]: Is there something specific about this chemistry where we want to check the length of the mapping, but don't the other maps?

File: test/test_mcs.py (hunk 2 above)
Comment (richardjgowers) [reply test/test_mcs.py]: it's a little messy because you want to loop over fixtures, but I've changed it to that style now

**Pull request merged**